### ISSUE TITLE
`indent` rule does not apply to `Import/ExportDeclaration`

### ISSUE THREAD

**Issue by justinanastos** [eslint/eslint#8342]
**Tell us about your environment**

* **ESLint Version:** v3.18.0
* **Node Version:** v6.10.1
* **npm Version:** v3.10.10

**What parser (default, Babel-ESLint, etc.) are you using?**
default

**Please show your full configuration:**

**What did you do? Please include the actual source code causing the issue.**

```js
// eslint indent: ['error', 2]
import {
                                    a
} from 'b';

export {
                                    a
} from 'b';
```

**What did you expect to happen?**
>Expected indentation of 2 spaces but found 4. (indent)

**What actually happened? Please include the actual, raw output from ESLint.**
No errors

**Comment by gyandeeps** [comment]
I think we never handled import export for indent. @not-an-aardvark is the re-write handling this?

**Comment by justinanastos** [comment]
Just incase it's not @gyandeeps , I submitted #8343 :)

**Comment by justinanastos** [comment]
I hadn't thought about `export` declarations. That can easily be added in too if this is something you'd be interested in.

**Comment by not-an-aardvark** [comment]
> I think we never handled import export for indent. @not-an-aardvark is the re-write handling this?

Yes.

**Comment by justinanastos** [comment]
After reading over https://github.com/eslint/eslint/pull/7618, it seems that this won't be necessary. This _may_ be a stopgap solution depending on how much work is left for https://github.com/eslint/eslint/pull/7618?

**Comment by not-an-aardvark** [comment]
Fixed by https://github.com/eslint/eslint/pull/7618
